src/SearchTest.java

```diff
@@ -0,0 +1,45 @@
+import static org.junit.Assert.*;
+
+import org.junit.Before;
+import org.junit.Test;
+public class SearchTest
+{
+	Search search;
+	int arr[]={-10,-2,0,1,5,8,15};
+	int to_search=5;
+	@Before
+	public void initializeClass()
+	{
+		search=new Search();
+	}
+	@Test
+	public void linearTest()
```

Comment:
```suggestion
	public void linearTest_keyPresentInArray()
```

Reply:
addes test case for key not present as well.